source/game/kart/KartObject.cc

```diff
@@ -118,9 +114,6 @@ const KartAccessor *KartObject::accessor() const {
 }
 
 KartObject *KartObject::Create(Character character, Vehicle vehicle, u8 playerIdx) {
-    Abstract::List list;
-    s_list = &list;
-
```

Comment:
Recall that `nw4r::ut::List_Init` clears the list. The same should be done here.